new-lamassu-admin/src/routing/routes.js

```diff
@@ -147,6 +149,13 @@ const tree = [
         route: '/compliance/customers',
         component: Customers
       },
+      {
+        key: 'blacklist',
+        label: 'Blacklist',
+        route: '/compliance/blacklist',
+        component: Blacklist
+      },
+      // what is this last one for
```

Comment:
What is this comment?

Reply:
That was a question I wanted to ask you but ended up forgetting. I started taking notes on the side instead of commenting, that one escaped my eye...